### DISCUSSION TITLE
All I need is some IMU data, could someone help me set up Airsim to achieve that?

### DISCUSSION THREAD

**Discussion by DominykasSimpukas** [microsoft/AirSim#4828] [Support Q&A]
Hello,
I'm electronics engineering student working on drone navigation estimation system, before testing my  hardware I want to try out it virtually, but I need some IMU data, I have found this very powerful tool and I think it is exactly what I need, the only problem is that I'm not IT or game developer, I have 0 knowledge in this kind of stuff, I tried to do it myself for a month, but no results, so maybe someone could help me set up Airsim to record more data? I have read modify_recording_data.md file, but it did not helped me, I'm still have no Idea what I'm doing and how to make this simulation running. I  need to build Airsim project (already done that), change code so it would include IMU data into record file and run it. I would be very thankful if some pro could help me sort this out and help me to move on.

**Comment by xxEoD2242** [comment]
Hey @DominykasSimpukas,

Send me an email at tylerfedrizzi@codex-labs-llc.com. AirSim is not longer supported by Microsoft, but we have continued and built an entire interface so you don't have to be a game dev to use the system. I'll get you setup with our system for what you need.

Our system: [SWARM Simulation Platform](https://www.swarmsim.io)

**Reply by DominykasSimpukas** [reply]
Hello, thanks for replying. I wrote you and email, we can continue there.
